openff/nagl/nn/_models.py

```diff
@@ -113,6 +116,47 @@ def _as_nagl(self):
 
 
 class GNNModel(BaseGNNModel):
+    """
+    A model that applies a graph convolutional step followed by
+    pooling and readout steps.
+
+    Parameters
+    ----------
+    convolution_architecture: Union[str, GCNStackMeta]
+        The graph convolution architecture.
+        This can be given either as a class,
+        e.g. :class:`~openff.nagl.nn.gcn._sage.SAGEConvStack`
+        or as a string, e.g. ``"SAGEConv"``.
+    n_convolution_hidden_features: int
+        The number of hidden features in the convolutional layers.
+    n_convolution_layers: int
+        The number of convolutional layers.
+    n_readout_hidden_features: int
+        The number of hidden features in the readout layers.
+    n_readout_layers: int
+        The number of readout layers.
+    activation_function: Union[str, ActivationFunction]
+        The activation function to use.
+        This can be given either as a class,
+        e.g. :class:`~openff.nagl.nn.activation.ActivationFunction.ReLU`,
+        or as a string, e.g. ``"ReLU"``.
```

Comment:
```suggestion
    activation_function: Union[str, ActivationFunction]
        The activation function to use for the readout module.
        This can be given either as a class,
        e.g. :class:`~openff.nagl.nn.activation.ActivationFunction.ReLU`,
        or as a string, e.g. ``"ReLU"``.
```